test/test.js

```diff
@@ -0,0 +1,49 @@
+var createS3UploadParams = require('../index').createS3UploadParams
+var reverse = require('../index').reverse
+var assert = require('assert');
+
+describe('Reverse', function () {
+    const tests = [
+        { word: 'abc', expected: 'cba' }
+    ];
+
+    tests.forEach(({ word, expected }) => {
+        it(`reverses ${word} correctly`, function () {
+            assert.equal(reverse(word), expected);
+        });
+    });
+});
+
+describe('CreateS3UploadParams', function () {
+    ip_ranges_example = {
+        "syncToken": "0123456789",
+        "createDate": "yyyy-mm-dd-hh-mm-ss",
+        "prefixes": [
+            {
+                "ip_prefix": "cidr",
+                "region": "region",
+                "network_border_group": "network_border_group",
+                "service": "subset"
+            }
+        ],
+        "ipv6_prefixes": [
+            {
+                "ipv6_prefix": "cidr",
+                "region": "region",
+                "network_border_group": "network_border_group",
+                "service": "subset"
+            }
+        ]
+    };
+    const tests = [
+        { data: 'syncToken: A123', expectedKey: '321-ipranges.json' },
```

Comment:
Codacy has a fix for the issue: [Replace `····{·data:·'syncToken:·A123',·expectedKey:·'321-ipranges.json'` with `{·data:·"syncToken:·A123",·expectedKey:·"321-ipranges.json"`](https://app.codacy.com/gh/dmgress/aws-ip-ranges/pullRequest?prid=9261990)
```suggestion
    { data: "syncToken: A123", expectedKey: "321-ipranges.json" },
```